tikv-client-proto/proto/backup.proto

```diff
@@ -74,7 +74,7 @@ message Schema {
     uint32 tiflash_replicas = 6;
 
     // stats represents the dump stats for a analyzed table, which generate by DumpStatsToJSON
-    // https://github.com/pingcap/tidb/blob/e136429d8dc5d70f43cd3f94179b0b9f47595097/statistics/handle/dump.go#L116
+    // `<https://github.com/pingcap/tidb/blob/e136429d8dc5d70f43cd3f94179b0b9f47595097/statistics/handle/dump.go#L116>`
```

Comment:
the proto files are copy-pasted from upstream kvproto, so I think we should ignore or exclude this directory rather than make changes.

Reply:
Well at first I tried to alter the way how proto files are process https://github.com/tikv/protobuf-build/pull/45/commits/bbe471f51a0d97fb0a00b41d145371f662e69f5d , but it seemed like a bit long way to go. So I took you advice and excluded the directory from the doc check.